cmd/link.go

```diff
@@ -67,7 +67,7 @@ Please delete service and recreate it using 'odo service create %s'`, serviceNam
 			// service is in running state.
 			// This can take a long time to occur if the image of the service has yet to be downloaded
 			_, err = client.WaitAndGetSecret(serviceName, projectName)
-			util.CheckError(err, "")
+			util.CheckError(err, "Waiting for secret of service %s to come up", serviceName)
```

Comment:
This message will be displayed only after it is done with waiting, and only if there was an error :-)
It should be just regular `fmt.Printf` just before WaitAndGetSecret

Reply:
OK, I misunderstood your previous comment :)